uportal-war/src/main/java/org/jasig/portal/portlets/portletadmin/PortletAdministrationHelper.java

```diff
@@ -253,13 +209,71 @@ public PortletDefinitionForm getPortletDefinitionForm(IPerson person, String por
         return form;
     }
 
+    /*
+     * Add groups and permissions to form.
+     */
+    private void getGroupPermissions(IPortletDefinition def, PortletDefinitionForm form, String activity) {
```

Comment:
I suggest a new name for this method.  A method that starts with "get" should return something.  Also "get" is so intimately associated with beans & properties that (IMHO) it's probably better not to use "get" to name a method that has any real behavior.  Maybe something like "addGroupPermissionsToForm."